feeder/util/mqtt/authentication.py

```diff
@@ -0,0 +1,42 @@
+import asyncio
+import re
+
+from hbmqtt.plugins.authentication import BaseAuthPlugin
+from feeder.database.models import KronosGateways
+from secrets import token_hex
+
+
+local_username = "local_%s" % token_hex(8)
+local_password = token_hex(16)
+
+
+class PetnetAuthPlugin(BaseAuthPlugin):
+    username_regex = re.compile(r'^/pegasus:(?P<gateway_id>.*)$')
+
+    def __init__(self, context):
+        super().__init__(context)
+
+    @asyncio.coroutine
```

Comment:
This wrapper is also redundant (I think 🤔)... Using async def is just the >Python 3.4 syntactic sugar for an async coroutine.